src/model_import.jl

```diff
@@ -0,0 +1,40 @@
+using YAML
+using Requests
+
+function guess_model_type(data)
+    if ("shocks" in keys(data["symbols"]))
+        if typeof(data["equations"]) == Dict{Any,Any}
+            return :dtcscc
+        else
+            return :dynare
+        end
+    else
+        return :dtmscc
+    end
+end
+
+function yaml_import(url)
+
+    if match(r"(http|https):.*", url) != nothing
+        res = get(url)
+        buf = IOBuffer(res.data)
+        data = YAML.load(buf)
+    else
+        data = YAML.load_file(url)
+    end
+
+    # fname = basename(url)
+
+    model_type = guess_model_type(data)
+
+    sym_model = SymbolicModel(data, model_type, "filename")
```

Comment:
Can we do better with the filename here? Shouldn't it be at the end of `url` just before `.yaml` or `.yml`?